contracts/MetaIdentityManager.sol

```diff
@@ -150,6 +150,8 @@ contract MetaIdentityManager {
         onlyOlderOwner(identity, sender)
         rateLimited(identity, sender)
     {
+        // an owner should not be allowed to remove itself
+        require(msg.sender != owner);
```

Comment:
Should this not be `require(sender != owner);` to allow meta tx?

Reply:
yep, thanks!